api/Document.json

```
@@ -7651,7 +7651,8 @@
               "version_added": "33"
             },
             "edge": {
-              "version_added": "18"
+              "version_added": "18",
+              "notes": "Before Edge 18, this event handler attribute was not supported, however the event itself was supported since Edge 12. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```

nitpick: punctuation near "however" is so particular
```suggestion
              "notes": "Before Edge 18, this event handler attribute was not supported; however, the event itself was supported since Edge 12. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
```

---

Might be relevant over on #6849, too